src/code/z_effect_soft_sprite_dlftbls.c

```diff
@@ -1,56 +1,34 @@
-#include "global.h"
-#include "initvars.h"
+#include "z64effect.h"
+#include "segment_symbols.h"
 
-#define EFFECT_SS_OVERLAY(name)                                                                \
-    {                                                                                          \
-        SEGMENT_ROM_START(ovl_##name), SEGMENT_ROM_END(ovl_##name), SEGMENT_START(ovl_##name), \
-            SEGMENT_END(ovl_##name), NULL, &name##_InitVars, 1,                                \
-    }
+// Init Vars and linker symbol declarations (used in the table below)
+#define DEFINE_EFFECT_SS(name, _enumValue) \
+    extern EffectSsInit name##_InitVars;   \
+    DECLARE_OVERLAY_SEGMENT(name)
```

Comment:
Any advantage or disadvantage to doing the init vars and linker together in 1 macro instead of having it split into 2 like OoT?

Reply:
I think it looks cleaner in the C file this way